src/lib/utils/hooks/SynapseAPI/oauth/useOAuthClient.ts

```diff
@@ -6,22 +6,16 @@ import {
   useQueryClient,
 } from 'react-query'
 import { SynapseClient } from '../../..'
-import { displayToast } from '../../../../containers/ToastMessage'
 import { SynapseClientError } from '../../../SynapseClientError'
 import { useSynapseContext } from '../../../SynapseContext'
 import { OAuthClient, OAuthClientList } from '../../../synapseTypes/OAuthClient'
 
-const oAuthQueryKeys = {
-  all: (accessToken: string) => ['oAuthClient', accessToken],
-  edit: (accessToken: string, id: string) => ['oAuthClient', accessToken, id],
-}
```

Comment:
Sorry if I was unclear, IMO it's ok to keep the `all` key (consider renaming it to `list`, since it's only used for listing OAuth Clients for now) and then invalidate that key. Just don't need the `edit` key, since it's not used.